Lib/test/test_future.py

```diff
@@ -175,7 +175,7 @@ def _exec_future(self, code):
         scope = {}
         exec(
             "from __future__ import annotations\n"
-            + code, {}, scope
```

Comment:
I removed this because if both `globals` and `locals` are passed to `exec`, it treats the code as it if were executed in a class definition, which makes the new test I've added harder than it needs to be (we can't access `c` so easily). Removing this doesn't seem to break any of the rest of the tests, so I _hope_ this is fine?